truenas/webui
# NAS-110907 / 21.06-BETA.1 / Small fixes for Permission Editor

Changes:
- Fix empty field for dataset path for ACL, POSIX ACL, Permission Editor forms
- Added types for updateCall
- Restrict subscriptions to take first result and unsubscribe while filtering in combobox
- Add debounce for keyup event for combobox
- Fix empty "User" and "Group" fields in combobox for ACL items

Testing
Open "Storage" page and choose "Edit Permissions" on dataset then switch to "ACL Manager" and make sure:
- Change "User" or "Group" and save it then refresh page and check if it's still valid
- Add a new ACL item with tag "Mask" and save it then refresh page and check if "Who" field is populated with "Mask"
- Add a new ACL item with tag "User", choose any user for "User" field save it then refresh page and check if "User" field is populated with username.
- Add a new ACL item with tag "Group", choose any group for "Group" field save it then refresh page and check if "Group" field is populated with groupname.
- "Dataset path" field shouldn't be empty
